proposals/000-comm-grants.md

```diff
@@ -0,0 +1,223 @@
+# Community Grants
+
+## Introduction
+
+This proposal is for the Haskell Foundation to undertake a direct small grant
+program, funded by individual donations to the foundation, to distribute funding
+where it can make a difference to the Haskell community.  The aim is to make the
+Haskell Foundation a more appealing target for individuals wishing to donate for
+the good of the entire Haskell community.  This indefinite ongoing program would
+accept submissions from the community on a rolling basis, and award up to $2000
+grants to individuals as a stipend to support their work on the proposed
+project.
+
+The amount of money involved is small, so the primary risk of the proposal is
+that funding decisions would generate discouragement or ill will from rejected
+submissions.  The risk can be partially mitigated by the use of a random lottery
+system to select winners after an initial selection of submissions that the
+Foundation believes are worth accepting.
```

Comment:
I like this idea! A random function seems to me a good way to avoid any negative sentiment. However, I would like to point out something that is possibly a nitpick surrounding how we talk about these sums of money. 

Throughout the conversation on discourse, as well as above, we refer to this amount of money as "small". This points to a slight disconnect in the way we're all framing this: $2000 is *alot* of money depending on whom you ask! If I were a grad student or an junior professional, I'd be chomping at the bit to engage this process and score a month or two's worth of rent for open source work, and I certainly would not expect this to be recurring - it's alot of money after all! I don't think there's much utility in framing this in terms of the size of the grant contributing to the risk factor. I think it might just be inherent to any conversation involving competition for sums of money.